ContactDetailsApi.Tests/DynamoDbTests.cs

```diff
@@ -0,0 +1,48 @@
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.DynamoDBv2.Model;
+using Amazon.XRay.Recorder.Core;
+using Amazon.XRay.Recorder.Core.Strategies;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace ContactDetailsApi.Tests
+{
+    [TestFixture]
+    public class DynamoDbTests
+    {
+        protected IDynamoDBContext DynamoDbContext { get; private set; }
+        protected AmazonDynamoDBClient DynamoDBClient { get; private set; }
+
+        [SetUp]
+        protected void RunBeforeTests()
+        {
+            AWSXRayRecorder.Instance.ContextMissingStrategy = ContextMissingStrategy.LOG_ERROR;
+            var clientConfig = new AmazonDynamoDBConfig { ServiceURL = "http://dynamodb-database:8000" };
+            DynamoDBClient = new AmazonDynamoDBClient(clientConfig);
+            try
+            {
+                var request = new CreateTableRequest("ContactDetails",
+                new List<KeySchemaElement> { new KeySchemaElement("targetId", KeyType.HASH), new KeySchemaElement("id", KeyType.RANGE) },
+                new List<AttributeDefinition> { new AttributeDefinition("targetId", ScalarAttributeType.S), new AttributeDefinition("id", ScalarAttributeType.S) },
```

Comment:
I would suggest moving "targetid" and "id" to variables like so 👍 
private const string TARGETID = "targetid", etc

Reply:
I've made the change